[ci] Add A100 queue into AWS CI template

khluu opened the pull request:
Enable steps defined in `test-pipeline.yaml` with key `gpu: a100` to run with A100 GPUs

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

simon-mo reviewed (approved):
* Can you add a simple test job running nvidia-smi
* Can you add priority class (in K8s?) so benchmark runs first?

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>

khluu commented:
I added a test job with A100 and also added priority class for both CI & perf benchmark following these priority classes definition: https://github.com/vllm-project/buildkite-ci/blob/main/infra-k8s/a100-priority-class.yaml

khluu committed:
p

Signed-off-by: kevin <kevin@anyscale.com>
